scripts/enrich_acmg_sf.py

```
@@ -0,0 +1,195 @@
+
+def process_acmg_sf_file(input_file: str, output_file: str, delay: float = 0.2):
+    """
+    Process the raw ACMG SF file and enrich it with gene identifiers.
+    
+    Args:
+        input_file: Path to the raw TSV file
+        output_file: Path to the output enriched TSV file
+        delay: Delay in seconds between API calls (to be respectful to the API)
+    """
+    # Column mapping from raw to final format
+    column_mapping = {
```

Ruff (0.14.14)

[error] Local variable `column_mapping` is assigned to but never used

Remove assignment to unused variable `column_mapping`

(F841)